libexec/scoop-cleanup.ps1

```diff
@@ -0,0 +1,80 @@
+# Usage: scoop cleanup <app> [options]
+# Summary: Update apps, or Scoop itself
```

Comment:
Should be `"Clean all or specific Scoop apps"`, instead of `"Update apps, or Scoop itself"`?